### ISSUE TITLE
Python requirements: Apple Silicon

### ISSUE THREAD

**Issue by joeflack4** [monarch-initiative/mondo-ingest#381]
## Overview
Nico and I are fortunate enough to be getting some new beastly Macbooks that use Apple Silicon processors. Mine just came in and to no surprise, `pip install -r python-requirements.txt` did not work in `mondo-ingest`.

### Non-poetry/tox environments
For `mondo-ingest` at least, there is another file called `python-requirements-unlocked.txt`, which simply lists the packages that we're interested in installing, usually version agnostic, and does not contain any of the transitive dependencies for those packages.  Installing from this worked for me.

### Poetry/tox environments
Not sure yet; haven't thought about or researched this yet. @hrshdhgd Any ideas?

## Repos where changes needed
- [ ] `mondo-ingest` (addressed by #380)
- [ ] `mondo`
- [ ] `mondolib`
- [ ] more?

**Comment by hrshdhgd** [comment]
This is a good time to implement poetry in my opinion. The multiple requirements files are confusing in my opinion

**Comment by hrshdhgd** [comment]
.. and `mondolib` does not need any updates. It's already build using the cookiecutter.

**Comment by matentzn** [comment]
I agree with standardising, and moving everything to the same build/dependency systems
